joxxe274/pac-man-react
# feat: #1 🐣 Add pac-man component

## feat: #1 🐣 Add pac-man component and unit tests

This pull request adds the `Pac-man` component and its unit tests to the project.

### Changes Proposed
- Add the `Pac-man` component
- Add unit tests for the `Pac-man` component

### How to test the changes?
- Verify that the `Pac-man` component is correctly rendered on the page
- Verify that the unit tests for the `Pac-man` component pass
